src/legacy/core_plugins/kibana/public/discover/components/doc/doc.test.tsx

```diff
@@ -25,7 +25,7 @@ import { findTestSubject } from '@elastic/eui/lib/test';
 import { Doc, DocProps } from './doc';
 
 jest.mock('../doc_viewer/doc_viewer', () => ({
-  DocViewer: 'test',
+  DocViewer: () => null,
```

Comment:
If the doc viewer is actually rendered, `test` throws another warning because it's not a valid react component